async-google-apis-common/Cargo.toml

```diff
@@ -27,3 +27,7 @@ serde_json = "~1.0"
 tokio = { version = "1.0", features = ["fs"] }
 tokio-stream = "^0.1"
 yup-oauth2 = "~5"
+
+[features]
+default = ["multi-thread"]
+multi-thread = []
```

Comment:
`default` and `multi-thread` feature are doing the same thing now, could you remove the `[features]` section and respective feature-guards added in `lib.rs` and `http.rs`?

Reply:
Done